common/app/common/NewNavigation.scala

```diff
@@ -382,21 +375,25 @@ object NewNavigation {
       SectionsLink("au/lifeandstyle/health-and-wellbeing", healthAu, Life)
     )
 
-    def getSectionLinks(sectionName: String, edition: Edition): Seq[NavLink] = {
+    def getSectionLinks(sectionName: String, edition: Edition): Tuple2[Seq[NavLink], Seq[NavLink]] = {
```

Comment:
Yes good shout. I thought about that after as well. I have it in my list to spend a few days next week refactoring - just need to get the rollout happening first 😓